main.go

```diff
@@ -366,12 +366,16 @@ func (s *DB) Scan(dest interface{}) *DB {
 
 // Row return `*sql.Row` with given conditions
 func (s *DB) Row() *sql.Row {
-	return s.NewScope(s.Value).row()
+	scope := s.NewScope(s.Value)
+	defer scope.trace(NowFunc(), false)
+	return scope.row()
 }
 
 // Rows return `*sql.Rows` with given conditions
 func (s *DB) Rows() (*sql.Rows, error) {
-	return s.NewScope(s.Value).rows()
+	scope := s.NewScope(s.Value)
+	defer scope.trace(NowFunc(), false)
```

Comment:
`Rows` returns multiple records, but in this case `scope.db.RowsAffected` cannot increment.
So I think it's better to remove it.